documentation/docs/protocol_specification/components/congestion_control.md

```diff
@@ -1,87 +1,186 @@
+Every network has to deal with its intrinsic limited resources in terms of bandwidth and node capabilities (CPU and
+storage). In this document, we present a congestion control algorithm to regulate the influx of messages in the
+network with the goal of maximizing throughput (messages/bytes per second) and minimizing delays. Furthermore, the
+following requirements must be satisfied:
 
-*   _Consistency_. If a message is written by one honest node, it shall be written by all honest nodes within some delay bound.
-*   _Fairness_. Nodes can obtain a share of the available throughput depending on their access Mana. Throughput is shared in such a way that an attempt to increase the allocation of any node necessarily results in the decrease in the allocation of some other node with an equal or smaller allocation (max-min fairness).
-*   _Security_. Malicious nodes shall be unable to interfere with either of the above requirements.
+* _Consistency_. If a message is written by one honest node, it shall be written by all honest nodes within some
+  delay bound.
+* _Fairness_. Nodes can obtain a share of the available throughput depending on their access Mana. Throughput is
+  shared in such a way that an attempt to increase the allocation of any node necessarily results in the decrease
+  in the allocation of some other node with an equal or smaller allocation (max-min fairness).
+* _Security_. Malicious nodes shall be unable to interfere with either of the above requirements.
 
-[![Congestion Control](/img/protocol_specification/congestion_control_algorithm_infographic.png)](/img/protocol_specification/congestion_control_algorithm_infographic.png)
+[![Congestion Control](/img/protocol_specification/congestion_control_algorithm_infographic_new.png)](/img/protocol_specification/congestion_control_algorithm_infographic_new.png)
 
 Further information can be found in the paper [Access Control for Distributed Ledgers in the Internet of Things: A Networking Approach](https://arxiv.org/abs/2005.07778).
 
 ## Detailed Design
 
-Our algorithm has three core components: 
-*   A scheduling algorithm which ensures fair access for all nodes according to their access Mana.
-*   A TCP-inspired algorithm for decentralized rate setting to efficiently utilize the available bandwidth while preventing large delays.
-*   A blacklisting policy to ban malicious nodes.
+Our algorithm has three core components:
+
+* A scheduling algorithm which ensures fair access for all nodes according to their access Mana.
+* A TCP-inspired algorithm for decentralized rate setting to efficiently utilize the available bandwidth while
+  preventing large delays.
+* An algorithm to check Time Since Confirmation of a tip, performed when creating a new message.
```

Comment:
I'm not sure this is the right place to introduce this. I would rather mention "a buffer management policy to deal with malicious flows" and move the TSC discussion to the TSA documentation